.fixtures.yml

```diff
@@ -10,6 +10,18 @@ fixtures:
       repo: git://github.com/puppetlabs/puppetlabs-yumrepo_core
       ref: 1.0.1
       puppet_version: ">= 6.0.0"
+    # Need by postgresql
+    augeas_core:
+      repo: git://github.com/puppetlabs/puppetlabs-augeas_core
+      ref: 1.0.4
+      puppet_version: ">= 6.0.0"
+    # Need by postgresql
+    concat:
+      repo: git://github.com/puppetlabs/puppetlabs-concat.git
+      ref: v6.0.0
+    postgresql:
+      repo: git://github.com/puppetlabs/puppetlabs-postgresql.git
+      ref: v6.0.0
```

Comment:
6.1.0 - be sure to check all the new additions that we are using the latest release

Reply:
This file has historically been the lower bound of version range, we test against the upper bound in `.fixtures-latest.yml`.